lib/kbuildbarn/options.go

```diff
@@ -69,13 +67,36 @@ func WithDirectoryUrlTemplate() Option {
 	return pathTemplateOption(DefaultDirectoryTemplate)
 }
 
-type templateArgsOption []interface{}
+type templateArgsOption [][]interface{}
 
 func (ta templateArgsOption) apply(opts *options) {
+	opts.TemplateArgs = ta[0]
+}
+
+func WithTemplateArgs(args []interface{}) Option {
+	return templateArgsOption{args}
```

Comment:
Maybe one slice dimension can be dropped if this is `return templateArgsOption(args)`?

...and then just `opts.TemplateArgs = ta` above?

Reply:
So fun fact, if you wrap []interface{} with a custom type,  it is now [][]interface{} (functionally).

so if on line 76 you `fmt.Println(args)` => `[ foo bar ]`

and on line 96, you fmt.Println(ta) => `[ [ foo bar ] ] `


*but* you can declare TA as an []interface{} (where the interface{} is another []interface{}. This requires a type check.

*or* you can declare ta as a [][]interface{}, and the go compiler knows that you can unwrap.

Reply:
Changed to just convert the []string input to []interface on the flag itself, easy fix.